src/utils.coffee

```diff
@@ -42,12 +44,26 @@ parse = (rawLink) ->
 # Read file to string
 readFile = (filename) ->
```

Comment:
I'd prefer to isolate the HTTP logic from the filesystem logic.

This could be done by changing the `inflate` function on [L83](https://github.com/jamesramsay/hercule/pull/30/files#diff-e7632c024955e4a1dc429d3253cbe6bcR83).